crates/oxc_linter/Cargo.toml

```diff
@@ -44,6 +44,7 @@ dashmap       = { workspace = true }
 convert_case  = { workspace = true }
 language-tags = { workspace = true }
 mime_guess    = { workspace = true }
+dunce = "1.0.4" # Normalize Windows paths to the most compatible format, avoiding UNC where possible
```

Comment:
Add this new dependency, but looks it will not increase the binary size, since it has been indirect imported before.

Reply:
This is because `oxc_resolver` already imports it.